server/config.js

```diff
@@ -10,7 +10,7 @@ const schema = {
   cookiePassword: joi.string().required(),
   sessionTimeoutMinutes: joi.number().default(30),
   apiGateway: joi.string().uri().default('http://localhost:3001'),
-  restClientTimeoutMillis: joi.number().default(5000)
+  restClientTimeoutMillis: joi.number().default(20000)

```

Comment:
It would be useful to expose the restClientTimeoutMillis as an env var and in the values.yaml in the same was as the api gateway https://github.com/DEFRA/mine-support-api-gateway/blob/master/server/config.js

Reply:
Updated.